go.mod

```diff
@@ -85,6 +38,7 @@ require (
 )
 
 replace (
+	github.com/prometheus/prometheus => github.com/prometheus/prometheus v0.0.0-20190818123050-43acd0e2e93f
```

Comment:
This was needed as for some reason the require changed, so I just changed it back to before. The change was also done in github.com/openshift/prom-label-proxy to match the deps.